backend/model/school.js

```diff
@@ -0,0 +1,58 @@
+const Student = require("./student");
+const Class = require("./Class")
+
+
+
+
+class School {
+  constructor() {
+    this.classes ={
+      
+    }
+   
+    
+  }
+
+
+   addClass(name, teacher) {
+    let newClass = new Class(name, teacher);
+    this.classes[name] = newClass;
+    return newClass
+  }
+
+
+  enrollStudent(className, student) {
+    let newStudent = new Student(student.name, student.age, student.city, student.grade)
+    if(this.classes[className]){
+      let enroll = this.classes[className]
+      enroll.students.push(newStudent)
+    }
+    let allStudents = this.classes[className]["students"]
```

Comment:
It looks like you are trying to add a new class and a new student at the same time. If the class doesn't exist it shouldn't be possible to add a new student